addons/l10n_ar_withholding/models/account_move.py

```diff
@@ -0,0 +1,12 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    l10n_ar_withholding_ids = fields.One2many(
```

Comment:
This was actually rightfully causing trouble in the tests.  The move_name on the account.move.line did not get computed correctly because of this.  Adding one2many with a domain like this has always been tricky and I think the right implementation might be to implement them as a compute.  @maq-adhoc @d-fence @jjscarafia

Reply:
https://github.com/odoo-dev/odoo/commit/64cde4dd85659e6563f164eb2048ec3d5dfbba03

Reply:
@jco-odoo  OK, fixed
https://github.com/odoo/odoo/compare/5e3e5f86675a460e11c22303a2b54fbb4816f294..5df59bfb0820791f6a0042e5c4140463bceab660